### DISCUSSION TITLE
Can I Use FrankenPHP as a Drop-in for FPM?

### DISCUSSION THREAD

**Discussion by splashsky** [php/frankenphp#278] [Q&A] [answered]
I have a couple of really old-school projects that don't rely on frameworks (or even really Composer :P) that I simply have FPM serve raw-dog through Caddy. If I were to build Caddy with this module, would I be able to simply have Franken serve these same apps through it's Caddyfile directive without "worker mode" so I don't have to change the entrypoints to my apps?

**Comment by dunglas** [comment]
Yes!

**Comment by EvanEdwards** [comment, marked as answer]
Yes.  That is how I am using it on a couple sites I have running under FrankenPHP.  It essentially functions about the same as Caddy+FPM in terms of simple configuration.  It just adds extra options like worker mode.
